reconfiguration/cmf/concord.cmf

```diff
@@ -247,6 +248,11 @@ Msg ClientReconfigurationStateReply 49 {
     list ClientStateReply states
 }
 
+Msg PruneSwitchModeRequest 50 {
+    uint64 sender_id
+    string configuration_json
```

Comment:
Who will compose this JSON? the operator?
Maybe it's better to stay in the CMF world and to define a new message for the configuration like
Msg PruneConfiguration{
        uint8 mode
        int rate
}

And then to compose it in that msg, this is how we work with other compositions.

Reply:
+1, I'd also prefer that - no need to mix CMF and JSON I believe

Reply:
IMO it couples message format with pruning mechanism. Let's say that there are 10 configurations of adaptive pruning. R we going to have 10 messages for each of them?

Reply:
You would have this logical coupling regardless of the format. If you use JSON, the coupling would simply be hidden as a string.